api/src/routing.js

```diff
@@ -110,6 +111,19 @@ app.postJsonOrCsv = (path, callback) => handleJsonOrCsvRequest('post', path, cal
 app.postJson = (path, callback) => handleJsonRequest('post', path, callback);
 app.putJson = (path, callback) => handleJsonRequest('put', path, callback);
 
+app.use(prometheusMiddleware({
+  metricsPath: '/api/v1/express-metrics',
+  metricsPrefix: 'cht_api',
+  // based on one-month analysed period of production traffic
+  durationBuckets: [
+    0.004, 0.007, 0.013, 0.027, 0.05,
```

Comment:
These buckets are weird at first glance. All the others double in size but these don't, even when you take rounding into account. Rounding to millis (3dp) yields:

```
    0.003, 0.006, 0.013, 0.025, 0.05,
```

Having said that, I'm not sure if there's much value in worrying about queries under 0.1s is there? Is there any downside to having such micro sized buckets?

Reply:
logic for the buckets is here: https://github.com/medic/cht-core/pull/8354#discussion_r1248249460. i'm happy to make adjustments, but that was thinking. give that a read and then let me know if your thoughts change?  i'm fine to eliminate micro-buckets if that seems valuable

Reply:
I read all of that but it doesn't explain where the numbers in the "proposed buckets" column in the spreadsheet came from (they're not calculations, just numbers). Am I missing something?

Reply:
ic. it sounds like you understand the "All Request Percentiles" column and the "Offline Replication Percentiles" column but not how they are combined.  that was kinda random and imprecise ... i interleaved and smoothed without keeping the work. 

so you likely aren't missing anything...  i'll change 0.027 to 0.025. if you have other suggestions can you propose as code changes?

~45% of requests are <0.1s so that is quite a big flattening

Reply:
```suggestion
    0.004, 0.007, 0.013, 0.025, 0.05,
```

Reply:
Ok that makes sense, and the code comment makes it clear. Thanks!